CHANGELOG.md

```diff
@@ -18,6 +19,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - False positive for subclass imports in `no-unused-imports` ([#845](https://github.com/pinterest/ktlint/issues/845))
 - False positive for static java function imports in `no-unused-imports` ([#872](https://github.com/pinterest/ktlint/issues/872))
 - Missing signature for KtLint CLI artifact published to Github release ([#895](https://github.com/pinterest/ktlint/issues/895))
+- Crash in annotation rule ([#868](https://github.com/pinterest/ktlint/issues/868))
```

Comment:
Should we close this issue?

Reply:
And probably worth to add this merged PR: https://github.com/pinterest/ktlint/pull/905